CONTRIBUTING.md

```diff
@@ -58,6 +58,44 @@ module.exports = function ModuleName(options,UI) {
 }
 ```
 
+Modules can be distributed independently via npm, yarn and cdn(for browser). `require` syntax should not be used if the module is intended to be used inside the browser.
```

Comment:
OK, let's say:

```
Image Sequencer is designed to be run either in the browser or in a Node.js environment. For dynamically loaded modules, that means that any uses of `require()` to include an external library must be compiled using a system like `browserify` or `webpack` to ensure browser compatibility. An example of this can be found here: 

https://URL_OF_BROWSERIFIED_EXTERNAL_MODULE

If you wish to offer a module without browser-compatibility, please indicate this in the returned `info` object as:

module.exports = [
  ModuleName,
  { only: ['node'] }
];

If you believe that full cross-compatibility is possible, but need help, please open an issue on your module's issue tracker requesting assistance (and potentially link to it from an inline comment or the module description). 
```

What do you think of that? This indicates a strong preference for cross-compatibility, with guidance for how to achieve it, but a workaround if you really cannot or you need help.

Reply:
This is awesome!! Putting this in now.

Reply:
@jywarren but for a node only module this may be imported from a json file, so then it'll be like 
```js
var info = require('./info')
```